SpatialGDK/Documentation/content/troubleshooting.md

```diff
@@ -58,7 +58,24 @@ This means you were using the GDK since pre-alpha. To fix the issue, delete the
 (The full error message looks like:
 `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\bin\Microsoft.Common.CurrentVersion.targets(1179,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=4.5" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [C:\MyProject\Game\Plugins\UnrealGDK\SpatialGDK\Build\Programs\Improbable.Unreal.Scripts\Build\Build.csproj]`)
 
-When you install Visual Studio as part of the [GDK dependencies]({{urlRoot}}/content/get-started/dependencies) set up step, ensure you have selected the **Universal Windows Platform development** workload. This workload includes .NET Framework 4.5. If you have already installed Visual Studio, you can add the missing workload by running the Visual Studio installer and clicking **Modify** on your existing Visual Studio Installation. 
+When you install Visual Studio as part of the [GDK dependencies]({{urlRoot}}/content/get-started/dependencies) set up step, ensure you have selected the **Universal Windows Platform development** workload. This workload includes .NET Framework 4.5. If you have already installed Visual Studio, you can add the missing workload by running the Visual Studio installer and clicking **Modify** on your existing Visual Studio Installation.
 
 ------
-_2019-05-02 Page updated._
\ No newline at end of file
+
+### Q: My worker instances are being disconnected from the SpatialOS Runtime unexpectedly while debugging locally.
+
+(You’ll see log messages similar to these while debugging your game: `LogNet: UNetConnection::Cleanup: Closing open connection. [UNetConnection] RemoteAddr: , Name: SpatialNetConnection_14, Driver: GameNetDriver SpatialNetDriver_17, IsServer: YES, PC: BP_PlayerController_C_0, Owner: BP_PlayerController_C_0, UniqueId: …`)
+
+This is caused by a [known issue](https://github.com/spatialos/UnrealGDK/issues/940) in the GDK for Unreal’s connection “heartbeating” logic that is used to disconnect worker instances that have stopped responding. 
+
+To work around the issue:
+
+1. In the Unreal Editor, navigate to **Edit** > **Project Settings** > **Runtime Settings**.
+1. Update the value of **Heartbeat Timeout** from the default 10 seconds to a much larger value that you’re unlikely to reach while debugging, such as 10000.
+1. **Navigate to Edit** > **Project Settings** > **Editor Settings**.
```

Comment:
```suggestion
1. Navigate to **Edit** > **Project Settings** > **Editor Settings**.
```

Should "10" and "1000" be in backticks as the user edits them (for consistency with below)?